### DISCUSSION TITLE
Error during installation

### DISCUSSION THREAD

**Discussion by ajazbi** [hezarai/hezar#164] [Q&A]
Hi,

I tried to install the Hezar on Anaconda powershell prompt but I get the following error:
----------------------------------------------------------------------------
Installing collected packages: tbb, intel-openmp, antlr4-python3-runtime, tensorboard-data-server, omegaconf, mkl, absl-py, torch, tensorboard, accelerate, hezar
  Attempting uninstall: tbb
    Found existing installation: TBB 0.2
ERROR: Cannot uninstall 'TBB'. It is a distutils installed project and thus we cannot accurately determine which files belong to it which would lead to only a partial uninstall.
------------------------------------------------------------------------------
I have already TBB 0.2, which version is needed? 

I tried on 2 laptops but no luck, Anybody can help? I

**Comment by arxyzan** [comment]
@ajazbi Hi, what OS did you try to install and what version of Hezar did this happen with?

**Comment by arxyzan** [comment]
@ajazbi I'm assuming that firstly, you are installing the CPU version of `torch` which requires the TBB package. Secondly, this does not seem to be specific to Hezar since all of our test suite running on GH Actions are indeed on CPU and every test passes before any release on PyPi. Can just try to install `pip install torch` and see if that works first?

**Reply by ajazbi** [reply]
Hi, I am using win10 with fresh anaconda installation. I Can not now make new envoironmet because I am in the middle of my project but I will test and report the result.
